web/html/src/manager/admin/config/monitoring-admin.js

```diff
@@ -109,7 +109,7 @@ const HelpPanel = (props) => {
         {t("The server uses ")}<a href="https://prometheus.io" target="_blank" rel="noopener noreferrer">{t("Prometheus")}</a>{t(" exporters to expose metrics about your environment.")}
       </p>
       <p>
-        {t("Refer to the ")}<a href="/docs/suse-manager/administration/monitoring.html" target="_blank">{t("documentation")}</a>{t(" to learn how to to consume these metrics.")}
+        {t("Refer to the ")}<a href={"/docs/" + window.docsLocale + "/suse-manager/administration/monitoring.html"} target="_blank">{t("documentation")}</a>{t(" to learn how to to consume these metrics.")}
```

Comment:
This one here is actually the only occurrence that does not use either `<HelpLink url=...>` or `<TopPanel helpUrl=...>` so I would not create a new component for just a single case. I can although look into adapting those two components so that they just take the e.g. `/reference/admin/task-engine-status.html` part.

With the downside that this would make `greping` for `/docs/` to find all of the `docs` links not work anymore.

Reply:
I changed the `<HelpLInk>` component as I said above. The only remaining case is the one you pointed out.